add note about selecting proper interface name for masquerading (#17433)
cilium-team/cilium: Documentation/gettingstarted/k8s-install-helm.rst

          ``tunnel=disabled`` from the helm command will configure Cilium to
          use overlay routing mode (which is the helm default).

>           Some linux distributions uses different interface naming convention.
>           If you use masquerade and option: ``egressMasqueradeInterfaces=eth0``
>           remember to replace eth0 with proper interface name.

Reviewer: ```suggestion
          Some Linux distributions use a different interface naming convention.
          If you use masquerading with the option ``egressMasqueradeInterfaces=eth0``,
          remember to replace ``eth0`` with the proper interface name.
```

Reviewer: Formatting
```
``eth0``
```
instead of 
```
eth0
```
will also address the report from the [documentation check](https://github.com/cilium/cilium/pull/17433/checks?check_run_id=3650417008).

I would also recommend addressing the warnings [from checkpatch](https://github.com/cilium/cilium/pull/17433/checks?check_run_id=3650418712) (shortening the commit object, and adding a commit description so we understand the motivation for the change when looking at the logs), although we don't strictly enforce this.
